Assets/_Game/Scripts/UI/ActivationPopupUI.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class ActivationPopupUI : MonoBehaviour
+{
+    [SerializeField]
+    private Animator anim;
+    [SerializeField]
+    private PlayerInput playerInput;
+
+    private void OnEnable()
+    {
+        if(playerInput != null)
+        {
+            playerInput.onControlsChanged += PlayerInput_onControlsChanged;
+
+            HandleInput(playerInput);
+        }
+    }
+
+    private void OnDisable()
+    {
+        if (playerInput != null)
+        {
+            playerInput.onControlsChanged -= PlayerInput_onControlsChanged;
+        }
+    }
+
+    private void Awake()
+    {
+        anim = GetComponent<Animator>();
+        playerInput = FindObjectOfType<PlayerInput>();
+    }
+
+    private void PlayerInput_onControlsChanged(PlayerInput input)
+    {
+        HandleInput(input);
+    }
+
+    public void HandleInput(PlayerInput input)
+    {
+        if (input.currentControlScheme == "Keyboard&Mouse") { ChangeUIPrompt("KEYBOARD"); }
```

Comment:
Advise changing both input.currentControlScheme (if you have access to it) and the parameter you're passing in to ChangeUIPrompt to an enum.

Reply:
Can't really modify the currentControlScheme but I made an enum for the ChangeUIPrompt.